configure.ac

```diff
@@ -176,7 +176,7 @@ PKG_CHECK_MODULES([libxml], [libxml-2.0])
 # don´t build the man pages
 if test "x$cross_compiling" = "xno"; then
   AM_CONDITIONAL([BUILD_MAN], [true])
-  DOXYGEN2MAN="\$(abs_builddir)/../doxygen2man/doxygen2man"
+  DOXYGEN2MAN="\"\$(abs_builddir)/../doxygen2man/doxygen2man\""
```

Comment:
For future-proofing it would be better to leave this alone and quote $(DOXYGEN2MAN) in docs/Makefile.am, otherwise if someone ever uses "$DOXYGEN2MAN" it will fail in a confusing way